lib/mcb/courses_editor_cli.rb

```diff
@@ -105,7 +84,23 @@ def ask_course_code
       end
     end
 
-  private
+    def multiselect(initial_items:, possible_items:)
```

Comment:
I think I agree with code climate on this one, it's a lot of nesting to comprehend in one go and I did get an "uhoh" feeling when I looked at it. Perhaps just factor out a method or two from the inner parts

Reply:
I've extracted a method but I'm not sure it's made it easier to read.